src/lib/config.js

```diff
@@ -44,13 +44,13 @@ const config = {
    * Checks arguments for specific (immediate action) flags and config
    * @param {Object} args The arguments passed in
    */
-  checkArgs: (args) => {
+  checkArgs: args => {
     // Show help
-    if (args.h) { output.log(config.helpMsg); process.exit(0); }
+    if (args.h) output.log(config.helpMsg); process.exit(0);
```

Comment:
Doesn't this exit the process every time? The conditional only applies to the log; the process.exit will get called every time this runs. Same with the next edit.